rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/interactor/CovidPatientRegisterActivityInteractor.java

```diff
@@ -45,13 +45,16 @@ protected Patient createPatient(String baseEntityId, JSONArray formFields) {
         String firstName = getStrField(formFields, CovidConstants.FormFields.PATIENT_FIRST_NAME);
         String lastName = getStrField(formFields, CovidConstants.FormFields.PATIENT_LAST_NAME);
         String sex = getStrField(formFields, Constants.DBConstants.SEX);
+        String patientAge = getStrField(formFields, Constants.DBConstants.AGE);
+        patientAge = StringUtils.isBlank(patientAge) ? "0" : patientAge;
+        String dob = getStrField(formFields, Constants.DBConstants.DOB);
```

Comment:
This class extends `PatientRegisterActivityInteractor` so this code is duplicated.

Could you maybe refactor this method so that the fields with equal values can be populated once in a protected method that can be called in each class?

i.e. `populateCommonPatientFields(Patient patient, JsonArray formFields)` and age, sex and dob can be populated here.